tempesta_fw/t/unit/test_sched_hash.c

```diff
@@ -43,96 +43,92 @@
 #include "sched_helper.h"
 #include "test.h"
 
-char req_str1[] = "GET / HTTP/1.1\r\nhost:host1\r\n\r\n";
-char req_str2[] = "GET / HTTP/1.1\r\nhost:host2\r\n\r\n";
-char req_str3[] = "GET / HTTP/1.1\r\nhost:host3\r\n\r\n";
-char req_str4[] = "GET / HTTP/1.1\r\nhost:host4\r\n\r\n";
-char *req_strs[] = {
-	req_str1,
-	req_str2,
-	req_str3,
-	req_str4,
+static char *req_strs[] = {
+	"GET / HTTP/1.1\r\nhost:host1\r\n\r\n",
+	"GET / HTTP/1.1\r\nhost:host2\r\n\r\n",
+	"GET / HTTP/1.1\r\nhost:host3\r\n\r\n",
+	"GET / HTTP/1.1\r\nhost:host4\r\n\r\n",
 };
-size_t req_strs_size = sizeof(req_strs) / sizeof(req_strs[0]);
 
-TEST(tfw_sched_hash, sg_empty)
+static TfwMsg * sched_hash_get_scheduler_arg (size_t connection_type);
```

Comment:
Space between the function name and the opening brace. This applies to multiple places in this patch set.
Also, please put the function name on a separate line. This also applies to multiple places.